docs/Reference/powerstates.md

```diff
@@ -2,32 +2,43 @@
-   | **4** | Sleep | 
-   | **5** | Power cycle |
-   | **7** | Hibernate |
-   | **8** | Power down/off |
-   | **10** | Reset |
-   | **12** | Soft power down/off |
-   | **14** | Soft reset |
-   | **100** | Power up to BIOS settings |
-   | **101** | Reset to BIOS settings |
-   | **104** | Reset to secure erase |
-   | **200** | Reset to IDE-R floppy disc |
-   | **201** | Power on to IDE-R floppy disc |
-   | **202** | Reset to IDE-R CD-ROM |
-   | **203** | Power on to IDE-R CD-ROM |
-   | **400** | Reset to PXE |
-   | **401** | Power on to PXE |
-
-Consider the current state of the system when implementing a possible action. For example: 
-
-* Reset to BIOS implies that the current system state is on or powered up.
-* Power up to BIOS implies that current system state is off or powered down.
-* Hibernate implies that the current system state is powered up. 
-
-If the system is already powered up, choosing to Power Up to BIOS will not have any effect on the system. A better choice is Reset to BIOS.
\ No newline at end of file
+Possible power actions are listed in the following table:
+
+   | Action #       | Power Action | [Start State^1^](#start-state) | Transition Description | ACPI State(s) | [In-band Agent Required^2^](#in-band-agent-required) | 
+   | :----------- | :------------------------ |   :------------------------ | :------------------------ | :------------------------ | :------------------------ |
+   | **2** | Power up/on | Powered down/off, Asleep, Hibernating | Power up/on fully | G0/S0 | No |
+   | **4** | Sleep (deep) | Powered up/on | Transition to a standby state of low-power usage and store system context (e.g., open applications) in memory | G1/S3 | Yes |
+   | **5** | Power cycle | Powered up/on | Transition to minimal power state and then power up/on fully | G2/S5 > G0/S0 |  No |
+   | **7** | Hibernate | Powered up/on | Transition to zero power usage and store system context in non-volatile storage | G1/S4 | Yes |
+   | **8** | Power down/off (hard) | Powered up/on | Transition to a fully powered down state | G2/S5 | No |
+   | **10** | Reset | Powered up/on | Perform hardware reset on the bus | N/A | No |
+   | **12** | Power down/off (soft) | Powered up/on | Transition to a very minimal power state | G2/S5 | Yes |
+   | **14** | Soft reset | Powered up/on | Perform a shutdown and then a hardware reset | N/A | Yes |
+   | **100** | Power up to BIOS settings | Powered down/off | Power to BIOS to verify or modify system configuration | G2/S5 | Yes  |
+   | **101** | Reset to BIOS settings | Powered up/on | Perform hardware reset on the bus to BIOS to verify or modify system configuration | G2/S5 | Yes  |
+   | **104** | Reset to secure erase | Powered up/on |  Perform hardware reset on the bus to secure erase, a process of completely erasing a solid state drive (SSD) | G2/S5 | Yes |
+
+## Alternative Boot Options
+
+Currently, the Toolkit doesn't natively support the 200 and 400-level calls.
```

Comment:
I thought the 400-level calls did work?  @bwendlandt-intel  can you confirm if these work?  I know the 200-level calls are not supported.

Reply:
So, it's just 200 that aren't supported then?

Reply:
@matt-primrose Do we support 104? Isn't RSE still in backlog, or is this different?

Reply:
@matt-primrose @bwendlandt-intel Let me know if I need to remove 104. I did place the 400s back in the first table.

Reply:
Done.